src/main/webapp/WEB-INF/web.xml

```diff
@@ -99,6 +99,33 @@
         </web-resource-collection>
     </security-constraint>
 
+    <security-constraint>
+        <web-resource-collection>
+            <web-resource-name>sd-document-creation</web-resource-name>
+            <description>SmartDocuments' Create Document callback</description>
```

Comment:
Minor thing but ss the `'` intended in the description? I would just leave that out.

Reply:
Removed the `